### PR TITLE
fix(new working example for drawer): surf1971

### PR DESCRIPTION
**Pull request by lalithkarikelli**
JIRA: SURF-1971

### LGTM's
* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: docs/components/drawer/index.html [hunk 1]
```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
         </footer>
       </div>
     </section>
+
+    <section>
+        <header>
+          <h2 id="basic-drawer">Positionable Descendants</h2>
```
Comment (100stacks) [inline docs/components/drawer/index.html:130]: Please update `id` from `basic-drawer` to `drawer-positionable-elements`.

File: docs/components/drawer/index.html [hunk 2]
```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
...
+                <label for="txtSlottedDrawerTitle">
+                  Title
+                </label>
+              </hx-text-control>
+              <p><code>hx-drawer &gt; [slot="fixed"]</code></p>
+            </form>
+          </header>
+  
+          <div>
+              <hx-disclosure aria-controls="drawer-with-slotted-positionable-children" class="hxBtn">
```
Comment (100stacks) [inline docs/components/drawer/index.html:151]: Instead of `children`, please change to `element`.

File: docs/components/drawer/index.html [hunk 3]
```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
...
+                  Drawer with Slotted Positionable Children
```
Comment (100stacks) [inline docs/components/drawer/index.html:152]: Instead of `children`, please change to `element`.

File: docs/components/drawer/index.html [hunk 4]
```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
...
+                </hx-disclosure>
+          
+                <hx-drawer id="drawer-with-slotted-positionable-children">
```
Comment (100stacks) [inline docs/components/drawer/index.html:155]: Instead of `children`, please change to `element`.

File: docs/components/drawer/index.html [hunk 5]
```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
         </footer>
       </div>
     </section>
+
+    <section>
+        <header>
+          <h2 id="basic-drawer">Positionable Descendants</h2>
+        </header>
```
Comment (100stacks) [inline docs/components/drawer/index.html:131]: Please update the header to:

```html
<header>
  <h2 id="drawer-positionable-elements">Drawer with Positionable Child Elements</h2>
  <p>
    <hx-icon type="info-circle"></hx-icon>
     Positionable elements (e.g., <code>&lt; hx-tooltip &gt;</code> and
     <code>&lt; hx-popover &gt;</code>) need to be direct children of the
     <code>&lt; hx-drawer &gt;</code> element, and placed in a named slot.
     Positionable elements require the <code>[slot="fixed]</code> attribute.
  </p>
</header>
```

File: docs/components/drawer/index.html (hunk 1 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html]: Changes have been done, please review.

File: docs/components/drawer/index.html (hunk 2 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html]: Changes have been done, please review.

File: docs/components/drawer/index.html (hunk 3 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html]: Changes have been done, please review.

File: docs/components/drawer/index.html (hunk 4 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html]: Changes have been done, please review.

File: docs/components/drawer/index.html (hunk 5 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html]: Changes have been done, please review.

File: docs/components/drawer/slotted-drawer-demo.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                    >
+                      <header></header>
+
+                      <hx-div ${this.attrBodyClass}>
+                        ...
+                      </hx-div>
+
+                      <footer>
+                        ...
+                      </footer>
```
Comment (100stacks) [inline docs/components/drawer/slotted-drawer-demo.js:41]: Please move this below the positionable elements.

```html
  <footer>
    . . .
  </footer>
</hx-drawer>
```

File: docs/components/drawer/slotted-drawer-demo.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+                    <hx-popover slot="fixed" id="popover1" position="left-middle">
+                      <header>
+                        Popover #1
+                      </header>
+            
+                      <hx-div>
+                        <p>This popover is a slotted child of hx-drawer.</p>
+                        <p><code>hx-drawer &gt; hx-popover[slot="fixed"]</code></p>
+                      </hx-div>
+                    </hx-popover>
```
Comment (100stacks) [inline docs/components/drawer/slotted-drawer-demo.js:58]: Please use 2 spaces of indentation for nesting elements. 

`<hx-tooltip>` and `<hx-popover>` should be indented to align with `<footer>`.

File: docs/components/drawer/index.html [hunk 8]
```diff
@@ -124,4 +125,128 @@ <h3 v-text="title"></h3>
...
     </section>
+
+    <section>
+        <header>
+            <h2 id="drawer-positionable-elements">Drawer with Positionable Child Elements</h2>
+            <p>
+              <hx-icon type="info-circle"></hx-icon>
+               Positionable elements (e.g., <code>&lt; hx-tooltip &gt;</code> and
+               <code>&lt; hx-popover &gt;</code>) need to be direct children of the
+               <code>&lt; hx-drawer &gt;</code> element, and placed in a named slot.
```
Comment (100stacks) [inline docs/components/drawer/index.html:136]: I made a mistake for this paragraph.  Please remove the extra spaces around the `<code>` blocks.

```html
<code>&lt;hx-tooltip&gt;</code>
<code>&lt;hx-popover&gt;</code>
<code>&lt;hx-drawer&gt;</code>
```

So, when rendered look like custom element tags:
`<hx-tooltip>`
`<hx-popover>`
`<hx-drawer>`

File: docs/components/drawer/slotted-drawer-demo.js (hunk 6 above)
Comment (lalithkarikelli) [reply docs/components/drawer/slotted-drawer-demo.js:41]: Changes have been done, please review.

File: docs/components/drawer/slotted-drawer-demo.js (hunk 7 above)
Comment (lalithkarikelli) [reply docs/components/drawer/slotted-drawer-demo.js:58]: Changes have been done, please review.

File: docs/components/drawer/index.html (hunk 8 above)
Comment (lalithkarikelli) [reply docs/components/drawer/index.html:136]: Changes have been done, please review.

**Pull request merged**